### PR TITLE
Development heroku

### PR DESCRIPTION
**Pull request by bibicollins**
-Deployment to heroku working with development branch. 
Buildpacks 
1.Ruby 
2.Python
3.Nodejs

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,8 +8,8 @@
 /coverage
 /client/coverage
 
-# production
-/build
+#process
+/client/build
```
Comment (elishkaflint) [inline .gitignore:12]: let's just check this works when we deploy via heroku!

Comment (bibicollins) [reply .gitignore:12]: Yep deployment still working!

**Review by elishkaflint — APPROVED** [review]
'i feel such a fool'
NOT ANYMORE
:ant:

**Review by laythq — APPROVED** [review]
Nice!

**Pull request merged**